inc/ix/ethdev.h

```diff
@@ -291,12 +295,9 @@ struct rte_eth_vmdq_mirror_conf {
  * A structure used to configure Redirection Table of  the Receive Side
  * Scaling (RSS) feature of an Ethernet port.
  */
-struct rte_eth_rss_reta {
-	/** First 64 mask bits indicate which entry(s) need to updated/queried. */
-	uint64_t mask_lo;
-	/** Second 64 mask bits indicate which entry(s) need to updated/queried. */
-	uint64_t mask_hi;
-	uint8_t reta[ETH_RSS_RETA_NUM_ENTRIES];  /**< 128 RETA entries*/
+struct ix_rte_eth_rss_reta_entry64 {
```

Comment:
This is the same definition as in DPDK
http://dpdk.org/doc/api/structrte__eth__rss__reta__entry64.html